src/bundle/Resources/translations/ibexa_admin_ui.en.xliff

```diff
@@ -76,6 +76,31 @@
         <target state="new">Select translation</target>
         <note>key: edit_translation.languages.select_language_title</note>
       </trans-unit>
+      <trans-unit id="e7b58c781e23a49ce354cdb479f14d6f75ca6e4f" resname="forgot_user_password.mail.message">
+        <source>We have received a request to reset the password for your account. Click "reset password" below to choose a new password:</source>
+        <target state="new">We have received a request to reset the password for your account. Click "reset password" below to choose a new password:</target>
+        <note>key: forgot_user_password.mail.message</note>
+      </trans-unit>
+      <trans-unit id="26ccb574b6cd1e49d40d32d9183db89065138be2" resname="forgot_user_password.mail.message_footer">
+        <source>If you did not request a password reset, please ignore this email, and your password will remain the same.</source>
+        <target state="new">If you did not request a password reset, please ignore this email, and your password will remain the same.</target>
+        <note>key: forgot_user_password.mail.message_footer</note>
+      </trans-unit>
+      <trans-unit id="a5ba92910b3dc61efc12c4f42cc869975be98723" resname="forgot_user_password.mail.message_title">
+        <source>Hello,</source>
+        <target state="new">Hello,</target>
+        <note>key: forgot_user_password.mail.message_title</note>
+      </trans-unit>
+      <trans-unit id="b7293d1f766a95b50cf3e076c5ef4cd623dc542c" resname="forgot_user_password.mail.reset_password">
+        <source>Reset password</source>
+        <target state="new">Reset password</target>
+        <note>key: forgot_user_password.mail.reset_password</note>
+      </trans-unit>
+      <trans-unit id="5fd68a8b7cdaa23771e0e970466b4432642df1d3" resname="forgot_user_password.mail.reset_your_password">
+        <source>Reset your password</source>
+        <target state="new">Reset your password</target>
+        <note>key: forgot_user_password.mail.reset_your_password</note>
+      </trans-unit>
```

Comment:
Those translations will be removed after the next execution of `php bin/console translation:extract -c ibexa_admin_ui`. Should be removed if they are not needed, or fixed.